index.html

```diff
@@ -34,9 +34,16 @@ <h2 class="centerh">Objectives/Goals</h2>
     </section>
   </article>
   <footer>
-    <div><h3>Contact Me</h3></div>
-    <div><p>Edgar Gasca Rodriguez<br><a href="/">(717)481-0806</p>
-      <p><a href="Github.com/edgaragascarodriguez">Github.com/edgaragascarodriguez</a><br><a href="edgargascarodriguez@gmail.com">edgargascarodriguez@gmail.com<a></div>
+    <div>
+      <h3>Contact Me</h3>
+      <h6>Edgar Gasca Rodriguez</h6>
+        <a href="/">(717)481-0806</a>
+      </div>
+      <div>
+        <a href="Github.com/edgaragascarodriguez">Github.com/edgaragascarodriguez</a>
+        <br>
```

Comment:
I don't think she wants us using HTML page break tags. I have no issue with them personally. But I'd try to replace it with a CSS property.

Reply:
I used one. I think your right though. I might change it to a list